packages/vm/tests/api/consensusBugs.ts

```diff
@@ -0,0 +1,59 @@
+const level = require('level')
+import { Transaction } from '@ethereumjs/tx'
+import VM from '../../lib'
+import { SecureTrie as Trie } from '@ethereumjs/trie'
+import { DefaultStateManager } from '../../lib/state'
+import Common from '@ethereumjs/common'
+import { Block } from '@ethereumjs/block'
+import tape from 'tape'
+
+tape('Consensus bugs', (t) => {
+  t.test(
+    'transaction 0x8420dbe268209c4e921bf6b78bc9fb959d9d9553fd16aacb9dd8cad955dbe729 should evaluate to right state root',
+    async (st) => {
+      const txData = {
+        nonce: '0x1a',
+        gasPrice: '0xba43b7400',
+        gasLimit: '0x493e0',
+        to: '0x9ca228250f9d8f86c23690074c2b96d5f5479f79',
+        value: '0x0',
+        data:
+          '0xa9059cbb000000000000000000000000bfe465e7eb5a2928b5bf22bef93ad06089dc6179000000000000000000000000000000000000000000000000006a94d74f430000',
+        v: '0x1c',
+        r: '0xc04f048766bea3b20dcea6d2fcaecc92f1a615d337606ab6eec2a1e8e1f27bfe',
+        s: '0x1c57e62653724560b06c95be750d958840db9fa44ffa777b6625b344bfadbbcf',
+      }
+      const common = new Common({ chain: 'mainnet', hardfork: 'chainstart' })
+      const tx = Transaction.fromTxData(txData, { common })
+      const trie = new Trie(level('/Users/jochem/Library/Ethereum/ethereumjs/mainnet/state/'))
```

Comment:
We don't have a `/Users/jochem/Library/Ethereum/ethereumjs/mainnet/state/` state database on CI so I will remove this test. 😄  Please keep a local copy in case you want to keep.